### PR TITLE
Commitizen file

### PR DESCRIPTION
**Pull request by SafaeELMIRI**

### CODE REVIEW

**Comment by barraponto** [comment]
Hi @SafaeELMIRI ! Thanks a lot for your contribution.
Seeing as we already have a commitizen suggestion, could you use your text to improve on the current one?
https://github.com/barraponto/usegit/blob/main/suggestions/commitizen.md

I am happy there is more interest in commitizen.

**Comment by SafaeELMIRI** [comment]
@barraponto Done

File: components/Counter.vue [hunk 1]
```diff
@@ -35,3 +35,5 @@ const counter = ref(props.count)
     </button>
   </div>
 </template>
+
+<!-- Comment -->
```
Comment (barraponto) [inline components/Counter.vue:39]:
```suggestion
```

File: components/Counter.vue [hunk 2]
```diff
@@ -34,4 +34,4 @@ const counter = ref(props.count)
       +
     </button>
   </div>
-</template>
+</template>
```
Comment (barraponto) [inline components/Counter.vue:37]:
```suggestion
</template>

```

**Comment by SafaeELMIRI** [comment]
Thank you Sir and apologies for the confusion over the counter.vue file!

**Pull request merged**